hypervisor/include/arch/x86/asm/guest/optee.h

```diff
@@ -10,6 +10,9 @@
 #include <asm/guest/vm.h>
 #include <asm/vm_config.h>
 
+/* If the RDI equals to this value, then this is a RETURN after FIQ DONE */
+#define OPTEE_RETURN_FIQ_DONE	0xBE000006UL
```

Comment:
Where does the magic # come from? If it is from a spec, please identify the source here.

Reply:
This is from OPTEE. Standard OPTEE command.